docs/docs/1.1.6/policies/health-check.md

```diff
@@ -143,3 +143,8 @@ HTTP health checks are executed using HTTP 2
   - if **`receive`** section won't be provided or will be empty, checks
     will be performed as "connect only" and will be marked as successful
     when TCP connection will be successfully established.
+
+## Matching
+
+`HealthCheck` is an [Outbound Connection Policy](how-kuma-chooses-the-right-policy-to-apply.md#outbound-connection-policy).
+You can only use `kuma.io/service` in the `destination` section.
```

Comment:
Does this mean that `kuma.io/service` is the only supported value for matching destination? If so, rewrite as

"The only supported value for `destinations.match` is `kuma.io/service`."

(assumes we're ok with the dot notation for nested fields?)

Reply:
fixed